docs/marketplace-and-submit-job-v2/marketplace-and-submit-job-v2.md

```diff
@@ -81,38 +81,226 @@ And you can share the script, data, docker images or whole job configuration on
+| `parameters`                     | Object, optional           | Specify name and value of all the referencable parameters that will be used in the job template. They can be referenced by **_$$paramName$$_**.  |
+| `prerequisites`                  | List, required             | List of `prerequisite`, specify docker image at least |
+| `tasks`                          | List, required             | List of `taskRole`, one task role at least |
+
+> prerequisite
+
+| Field Name                       | Schema                     | Description                              |
+| :------------------------------- | :------------------------- | :--------------------------------------- |
+| `protocol_version`               | String, optional           | Protocol version, If omitted, it will be **_v2_**|
+| `name`                           | String, in `^[A-Za-z0-9\-._~]+$` format, required | Name of the prerequisite |
+| `type`                           | String, required           | Type of the prerequisite, must be **_data_**, **_script_**, or **_dockerimage_** |
+| `version`                        | String, optional           | Version of the prerequisite              |
+| `contributor`                    | String, optional           | Contributor of the prerequisite          |
+| `description`                    | String, optional           | Description of the prerequisite          |
+| `uri`                            | String, required           | Reference URL                            |
+
+
+> taskRole
+
+| Field Name                       | Schema                     | Description                              |
+| :------------------------------- | :------------------------- | :--------------------------------------- |
+| `role`                           | String in `^[A-Za-z0-9._~]+$` format, required | Name for the task role, need to be unique with other roles |
+| `data`                           | String, optional           | Data that will be used in the task role, name reference of a prerequisite |
+| `script`                         | String, optional           | Script to be executed in the task role, name reference of a prerequisite |
+| `dockerimage`                    | String, optional           | Docker image to be used for the task role, name reference of a prerequisite |
+| `resource`                       | Object, required           | Resource required for the task role |
+| `minFailedTaskCount`             | Integer, optional          | Number of failed tasks to kill the entire job, null or no less than 1 |
+| `minSucceededTaskCount`          | Integer, optional          | Number of succeeded tasks to kill the entire job, null or no less than 1 |
+| `parameters`                     | Object, optional           | Specify name and value of all the referencable parameters that will be used in the task role. They can be referenced by **_$$paramName$$_**. |
```

Comment:
>_$$paramName$$_ [](start = 188, length = 15)

if I would like reference a parameter in job parameters section, how to do that?